src/components/ActionMenu/components/SecondaryAction/SecondaryAction.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../../../styles/common';
+
+$breakpoint: 1080px;
+
+.SecondaryAction {
+  // Hacks
+  // Eventually pull in button-base and modify
+  // stylelint-disable declaration-no-important
+  button {
```

Comment:
These are temporary hacks but the safer way to do this would be to import button-base and do some overrides